keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaTaskMonitorAgent.kt

```diff
@@ -49,6 +51,8 @@ class OrcaTaskMonitorAgent(
         .associate {
           it.subject to
             async {
+              // TODO: add service account to task tracking table
+              // authorizationSupport.withSpinnakerAuthHeaders(serviceAccount) {
```

Comment:
Some places have no user information to pass along, like here. In those cases, regardless of the approach, we'll need to refactor things to include that information so that we can build the headers.